ecosystem/sep-0006.md

```diff
@@ -1110,18 +1094,6 @@ The response should be a JSON object like:
       "min_amount": 0.1,
       "max_amount": 1000,
       "types": {
-        "bank_account": {
-          "fields": {
-              "dest": {"description": "your bank account number" },
-              "dest_extra": { "description": "your routing number" },
-              "bank_branch": { "description": "address of your bank branch" },
-              "phone_number": { "description": "your phone number in case there's an issue" },
-              "country_code": {
-                "description": "The ISO 3166-1 alpha-3 code of the user's current address",
-                "choices": ["USA", "PRI"]
-              }
-          }
-        },
```

Comment:
To confirm, we're removing this because the `bank_account` type is requesting the user's KYC info like phone number and bank account number right? And we want them to do this via SEP-12?

Don't we still need the client to provide the `type=bank_account` query parameter though, so the anchor knows to request bank account fields in the `GET /customer` response? If that is the case, then we need to do something like this:

```
"bank_account": {}
```

This signals that the type should be passed but there are no extra fields that need to passed in the transaction creation request.

Reply:
If we do it like described above, that would mean that we accept field named `bank_account` with empty body though, am I wrong? 
We still have `type` in the request, it should be used as before

Reply:
> If we do it like described above, that would mean that we accept field named bank_account with empty body though, am I wrong?

On what endpoint would anchors accept a `"bank_account": {}` key-value pair from wallet requests? I'm referring to the fact that `GET /info` responses should have all possible values of the `type` query parameter as keys. So we shouldn't remove the entire `bank_account` object, just the items within the object, since no additional parameters should be sent to the anchor other than the `type` parameter in that case.